src/driver/franka_plan_runner.cc

```diff
@@ -189,18 +188,20 @@ int FrankaPlanRunner::RunFranka() {
               "error recovery for Reflex mode: {}.",
               ce.what());
           comm_interface_->PublishDriverStatus(false, ce.what());
+          std::this_thread::sleep_for(std::chrono::milliseconds(1000));
         }
       } else if (current_mode != franka::RobotMode::kIdle) {
-        auto err_msg {
-            fmt::format("RunFranka: robot cannot receive commands in mode: {}",
-                        utils::RobotModeToString(current_mode))};
+        auto err_msg {fmt::format("robot cannot receive commands in mode: {}",
+                                  utils::RobotModeToString(current_mode))};
         dexai::log()->error("RunFranka: {}", err_msg);
         comm_interface_->PublishDriverStatus(false, err_msg);
+        std::this_thread::sleep_for(std::chrono::milliseconds(1000));
       } else if (current_mode == franka::RobotMode::kUserStopped) {
         dexai::log()->error("RunFranka: robot is in User-Stopped mode");
         comm_interface_->PublishBoolToChannel(
             utils::get_current_utime(),
             comm_interface_->GetUserStopChannelName(), true);
+        std::this_thread::sleep_for(std::chrono::milliseconds(1000));
```

Comment:
Can we reduce sleep duration to something like `100 ms`? 1 second is a noticeable time delay for status to change after user un-estops the robot

Reply:
sure